src/picongpu/include/particles/pusher/particlePusherAxl.hpp

```diff
@@ -37,7 +37,11 @@ namespace picongpu
         template<class Velocity, class Gamma>
         struct Push
         {
-
+            /* this is an optional extention for sub-sampling pushes that enables grid to particle interpolation
```

Comment:
optional extension sounds to a (non-substepping) pusher implementor as if these typedefs can be neglected, which is not the case.

I would suggest something like:
```c++
/* Declaration of margins for accessing additional cells during the push
 *
 * In case of pushers using sub-stepping, additional interpolation from positions not along a
 * direct path between start/end of the step might necessary. This needs additional access to
 * surrounding cells and this option tells how many need to be accessed in maximum.
 *
 * If this is not a pusher that will perform additional access to field data, the margins are
 * probably 0.
 */